bitnami/mongodb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb
-version: 7.10.12
+version: 7.10.13
```

Comment:
I would bump a minor version because we are modifying the behavior of a value.

Reply:
done with last version of master